be/src/runtime/runtime_state.h

```diff
@@ -67,6 +67,8 @@ class QueryContext;
 // query and shared across all execution nodes of that query.
 class RuntimeState {
 public:
+    // for ut only
+    RuntimeState() {}
```

Comment:
if it was only used in UT. suggest make it private

Reply:
just follow the lines below, if it make private, it not visible to UT. friendly class is bad mechanism because of its abstraction leak.

Reply:
we have add flags -fno-access-control for UT.